net/pbr/Makefile

```diff
@@ -0,0 +1,198 @@
+# Copyright 2017-2022 Stan Grishin (stangri@melmac.ca)
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pbr
+PKG_VERSION:=0.9.9
+PKG_RELEASE:=24
+PKG_LICENSE:=GPL-3.0-or-later
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.ca>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/pbr/default
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=VPN
+	PROVIDES:=pbr
+	TITLE:=Policy Based Routing Service
+	URL:=https://docs.openwrt.melmac.net/pbr/
+	DEPENDS:=+ip-full +jshn +jsonfilter +resolveip
+	CONFLICTS:=vpnbypass vpn-policy-routing
+	PKGARCH:=all
+endef
+
+define Package/pbr
+$(call Package/pbr/default)
```

Comment:
Why this package provides itself based from ``define Package/pbr/default``?

Reply:
What's a better way to organize two packages: `pbr` and `pbr-iptables` from the same Makefile?